docs/reference/ilm/actions/ilm-allocate.asciidoc

```diff
@@ -7,16 +7,16 @@ Phases allowed: warm, cold.
 Updates the index settings to change which nodes are allowed to host the index shards
 and change the number of replicas.
 
+The allocate action is not allowed in the hot phase. 
+The initial allocation for the index must be done manually or via index templates.
+{ilm-init} doesn't handle index allocation during the hot phase.
```

Comment:
I could be missing something, but is this sentence needed? Seems like a restatement of the first one in the para. Feel free to leave in if I'm wrong.
```suggestion
```